.github/stale.yml

```diff
@@ -0,0 +1,30 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an issue or PR is considered stale
+daysUntilStale: 30
+
+# Number of days of inactivity before a stale issue or PR is closed
+daysUntilClose: 30
+
+# Issues or PRs with these labels will never be considered stale
+exemptLabels:
+  - up for grabs
+
+# Label to use when marking an issue or PR as stale
+staleLabel: waiting
+
+# Comment to post when marking an issue or PR as stale
+markComment: >
+  Hi all! This thread has not had any recent activity.
+  Can anyone comment on what's blocking progress?
```

Comment:
This line feels a bit weird. I would rather change it to "Any updates on this ?'